index.bs

```diff
@@ -532,7 +476,7 @@ in [[#es-extended-attributes]].
         IncludesStatement
 </pre>
 
-<div class="example">
+<div class="example" id="example-25f60a7c">
```

Comment:
Can we give these examples real IDs? I care about this a good bit so if you can't be bothered I guess I'll volunteer :)

Reply:
These are the current IDs, for better or worse. I guess I don't feel too strongly about keeping these IDs working, but folks might be linking to them. (I haven't checked, but either this is disabled or Bikeshed must generate a lot of warnings for the document in its current state.)

Reply:
Oh, that turns this from a "we should fix before merging" to a "we should maybe fix eventually".